src/hexen/g_game.c

```diff
@@ -2049,7 +2057,10 @@ void G_InitNew(skill_t skill, int episode, int map)
     }
 
     // Set up a bunch of globals
-    if (!demoextend)
+    // [crispy] since demoextend is the default, we also want to check to
+    // make sure we're not playing a demo
+    if ((!demoextend)
```

Comment:
No need for double parentheses here. Could you just add the extra conditions into the same line?